### DISCUSSION TITLE
combine work width

### DISCUSSION THREAD

**Discussion by rohne83** [Courseplay/courseplay#7209] [Q&A] [answered]
Ich versuche zur Zeit die Fortschritt E512 und E514 mods zu überarbeiten. Eins der großen Probleme die ich dabei hab, ist dass CP bei den 4.2m Schneidwerken (aber nur bei diesen) die Arbeitsbreite immer erstmal auf 5.5m setzt. Die ganzen Koordinaten bei workAreas etc scheinen aber soweit alle zu stimmen. Daher Frage, woher genau CP sich diese Information zieht.

**Comment by Tensuko** [comment, marked as answer]
Es wäre natürlich gut, wenn es in der mod richtig hinterlegt ist, zur not kannst du das auch in der vehicle config hinterlegen.
...\My Games\FarmingSimulator2019\modsSettings\Courseplay\VehicleConfigurations.xml

Oder du schaust dir mal toolManager.lua ab Zeile 463 an.

@pvaiko kann das vlt noch etwas besser erklären.

**Reply by pvaiko** [reply]
Debug level 21 zeigt an wo die 5.5 m kommt, stimmen die AI Marker?

**Reply by rohne83** [reply]
Danke euch, das hat mir sehr geholfen. Am Ende lag es daran dass die workArea des Strohhäckslers breiter war als die des Schneidwerks, und sich die Arbeitsbreite entsprechend von dort gezogen wurde.
